azure-pipelines.yml

```diff
@@ -85,6 +85,18 @@ jobs:
                   JHI_APP: webflux-mongodb-oauth2
                   JHI_ENTITY: none
                   JHI_PROTRACTOR: 1
+              webflux-couchbase:
+                  JHI_APP: webflux-couchbase
+                  JHI_ENTITY: none
+                  JHI_PROTRACTOR: 1
+              webflux-couchbase-session:
+                  JHI_APP: webflux-couchbase-session
+                  JHI_ENTITY: none
+                  JHI_PROTRACTOR: 1
+              webflux-couchbase-oauth2:
+                  JHI_APP: webflux-couchbase-oauth2
+                  JHI_ENTITY: none
+                  JHI_PROTRACTOR: 1
```

Comment:
We are above the limit of 10 here, so it will increase the build time.

The best would be to mix with the other webflux builds, and add new builds at https://github.com/hipster-labs/jhipster-daily-builds so these config will be tested each day. So we'll stay with 10 builds.

Reply:
@pascalgrimaud I added a bunch for reactive tests and [two more for reactive gateway](https://github.com/jhipster/generator-jhipster/pull/11223). I could move them, but I prefer reactive to be a first-class citizen.

Reply:
I was suggesting this because Couchbase is not used a lot. So launch Couchbase + Reactive builds for each PR is not really usefull.

We need to remember that the more builds we have, the more random failures... So restarting pull request can become a pain.

We have more and more options: testing all options for each PR is impossible. We need to make a choice. That's why the Daily Builds project is here ! :-)

Anyway, what you did is a good idea : testing these configs during the PR, when it's in Work In Progress, then once the PR is merged, we can move it to Daily Builds.

Reply:
This is exactly why I added the tests here, as soon as you validate the PR, I'll keep just one, or if you prefer I'll remove the 3 tests.

I'll do the PR for daily builds ASAP

Reply:
ok then. Can you just resolve the problem with lint plz ?

Reply:
Sorry, I missed it! Just fixed it